Cargo.toml

```diff
@@ -22,6 +22,7 @@ asm = []
 full-quickcheck = []
 
 [dependencies]
+ieee754 = "0.1"
```

Comment:
Nope, not at all. I mistakenly thought that that was the highest version number.